vision_agent/tools/tools.py

```diff
@@ -290,6 +290,14 @@ def _apply_object_detection(  # inner method to avoid circular importing issues.
             )
             function_name = "florence2_object_detection"
 
+        elif od_model == ODModels.AGENTIC:
+            segment_results = agentic_object_detection(
+                prompt=prompt,
+                image=segment_frames[frame_number],
+                fine_tune_id=fine_tune_id,
+            )
+            function_name = "florence2_object_detection"
```

Comment:
for reporting also update
```suggestion
            function_name = "agentic_object_detection"
```

Reply:
:o good catch, thank you.